app-text/master-pdf-editor: new ebuild

jpataias opened the pull request:
Add latest version of Master PDF Editor.

> Master PDF Editor is the optimal solution for editing PDF files in Linux. It enables you to create, edit, view, encrypt, sign and print interactive PDF documents.

In collaboration with @jfmcarreira.

monsieurp commented:
Thanks for the PR. @ulm what's your take on the licence of this software?

ulm commented:
> Thanks for the PR. @ulm what's your take on the licence of this software?

AFAICS, no problem adding this to the tree. However, the license
doesn't grant the right to redistribute, so the ebuild will need
mirror restriction.

(Also, about the license file itself: Please rewrap these long lines
to a reasonable length, like 70 chars. At the end of the file a
newline is missing.)

jpataias commented:
Added the required changes do ebuild and license file.

monsieurp commented on app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild:
```diff
@@ -0,0 +1,72 @@
+}
+
+pkg_postinst() {
+	sed -e '/^Exec=/s/masterpdfeditor4/masterpdfeditor4 %f/' -i /usr/share/applications/masterpdfeditor4-${PN}.desktop
```
Add a `|| die` to the end of the line here please.

monsieurp commented on app-text/master-pdf-editor/metadata.xml:
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>joaompssantos@gmail.com</email>
```
You will maintain this package. Please file a bug about this package to make sure your email in recorded in Bugzilla. Also, add the proxy-maintainer project. Developers of this project will proxy your changes.

jpataias replied:
Should I go to bugs.gentoo.org and a new report under Gentoo Linux --> New packages?

monsieurp replied:
Yes.

ulm commented on app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild:
```diff
@@ -0,0 +1,73 @@
+
+SRC_URI="http://get.code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+RESTRICT="mirror"
```
Small nitpick: The usual convention is for RESTRICT to go to the end of the second variables block (i.e., LICENSE, SLOT, KEYWORDS, [IUSE], [RESTRICT]).

jpataias replied:
Will do, thanks!

jpataias commented:
Corrected required changes and added a bug report: https://bugs.gentoo.org/635690.

Apparently, this was already done before for a different version: https://bugs.gentoo.org/610948. However I can't find the package on Gentoo tree or on the overlays.
